third_party/terraform/tests/resource_compute_region_health_check_test.go.erb

```diff
@@ -0,0 +1,596 @@
+<% autogen_exception -%>
```

Comment:
Instead of adding a whole bunch of new handwritten tests, would it be possible to make some or all of these generated?